### PR TITLE
LPS-130552 | master

### PR DESCRIPTION
**Pull request by ozysouza**
Rebased from #[6886](https://github.com/liferay-echo/liferay-portal/pull/6886)

https://issues.liferay.com/browse/LPS-130552

@david-gutierrez-mesa 
@Tim-Cao

### CODE REVIEW

**Comment by ozysouza** [comment]
Hi @Tim-Cao, I updated the pull request according your comments, but when I was writing the testcase about searchWebContent I noticed some things that would be good to change then I did some extra changes on the macro. Would you mind have a look at?

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro [hunk 1]
```diff
@@ -542,8 +542,36 @@ definition {
 		AssertClick(
 			locator1 = "WCD#WEB_CONTENT_TEXT_LINK_URL",
 			value1 = "${linkName}");
+	}
+
+	macro viewMedia {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro:501]: Merge this one into viewContent

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro [hunk 2]
```diff
@@ -415,6 +415,28 @@ definition {
 			value1 = "Undo");
 	}
 
+	macro view {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/webcontentdisplay/WebContentDisplayPortlet.macro:342]: Rename to viewContent

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, Could you update according to comments? Btw, break this large PR into multiple smaller PRs to avoid conflict, thanks.

**Comment by ozysouza** [comment]
ci:close

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update according to comments? Btw, break this large PR into multiple smaller PRs to avoid conflict, thanks.

Hi @Tim-Cao, okay! I've sent two new PR with some commits, when these one be merged I will send the remaining.

**Pull request closed without merging**